src/webgpu/api/validation/queue/destroyed/query_set.spec.ts

```diff
@@ -1,8 +1,7 @@
 export const description = `
 Tests using a destroyed query set on a queue.
 
-- used in {resolveQuerySet, timestamp {compute, render, non-pass},
-    pipeline statistics {compute, render}, occlusion}
+- used in pipeline statistics {compute, render}
```

Comment:
This could just be "TODO: Test with pipeline statistics queries as well." and the TODO at the bottom removed.